application-forum-ui/src/main/resources/ForumCode/ConfigurationAdmin.xml

```diff
@@ -39,21 +39,23 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{include reference="Licenses.Code.VelocityMacros"/}}
-
+  <content>{{include reference='Licenses.Code.VelocityMacros'/}}
```

Comment:
We usually maintain the elder style/code from the files. I think that the policy is to create a new commit for refactoring process (but it's not the case here since we're talking about quotes). Am I wrong, @oanalavinia?

Reply:
Indeed, usually a commit should be kept clean, to contain only what fixed the issue, and to have a separate one for refactoring, since it's easier for review